CONTRIBUTING.md

```diff
@@ -32,9 +56,54 @@ Please take a look at the current open issues to see if any of them interest you
 5. Describe the problem and add references to the related issues that the request solves.
 6. Request review of your code by other contributors. Try to improve your code as much as possible to lessen the burden on others.
```

Comment:
Should we also mention our convention about adding entries to `CHANGELOG.md` within pull requests?

Reply:
Yeah, that's a good point. I will add that.